### PR TITLE
Add kafka contract cleanup script

### PR DESCRIPTION
**Pull request by Cali0707**
This is a possible fix to the serverless upgrade test failures by calling a bash script to clean things up during the downgrade stage of the tests

## Proposed Changes

-
-
-

### CODE REVIEW

File: test/upgrade/installation/serverless.go [hunk 1]
```diff
@@ -251,3 +260,16 @@ func setStorageToAlpha(ctx *test.Context, name string) error {
 		return err
 	})
 }
+
+func downgradeKafkaContractsWithScript() error {
+	c := exec.Command("bash", "-s", "-", "kafka-broker-brokers-triggers", "kafka-channel-channels-subscriptions", "kafka-sink-sinks", "kafka-source-dispatcher-0", "kafka-source-dispatcher-1")
```
Comment (creydr) [inline test/upgrade/installation/serverless.go:265]: Maybe only one thing for the script: Should we include the names of the main configmaps already in the script, so support would not need to know the names of the CMs?
And maybe we can pass the namespace as a parameter to the script, so we could also use this (in the rare case) of using the namespaced-broker.

Comment (Cali0707) [reply test/upgrade/installation/serverless.go:265]: Yeah we could change that!

Comment (creydr) [reply test/upgrade/installation/serverless.go:265]: Sry, I could have responded earlier: #2555

**Comment by Cali0707** [comment]
Closing since we have #2554 merged (thanks @creydr !!!)

**Pull request closed without merging**